src/nvim/po/de.po

```diff
@@ -19,132 +19,375 @@ msgstr ""
 "Content-Type: text/plain; charset=ISO_8859-1\n"
 "Content-Transfer-Encoding: 8-bit\n"
 
-#: ../api/private/helpers.c:197
-msgid "Unable to get option value"
+#: ../api/buffer.c:170 ../api/buffer.c:281
+#, fuzzy
+msgid "Index out of bounds"
+msgstr "Index out of bounds"
+
+#: ../api/buffer.c:186
+msgid "Line index is too high"
+msgstr "Zeilenindex ist zu gro�"
+
+#: ../api/buffer.c:289
+msgid "Argument \"start\" is higher than \"end\""
+msgstr "Argument \"start\" is gr��er als \"end\""
+
+#: ../api/buffer.c:304
+msgid "All items in the replacement array must be strings"
+msgstr "Alle List-Eintr�ge in Ersatzarray m�ssen Zeichenfolgen sein"
```

Comment:
@mschilli87 thanks for the review, if you ask why it's because i based it off an existing translation. I'm not a German native speaker so i'll take your suggestions as gospel :)